lib/rules/new-line-before-expect.js

```diff
@@ -40,3 +47,13 @@ module.exports = function (context) {
     }
   }
 }
+
+/**
```

Comment:
@surdu , There is one more thing to fix here:  This function annotation makes no sense. Either delete it all together or correct expected input and return types. 
And squash it after ...
Sorry I didn't catch it first time around